topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.md

```diff
@@ -0,0 +1,643 @@
+
+TCGA Pancancer has uniformly processed Multi-omics data including RNA-Seq, copy number and mutational data. It covers 33 different cancer types and having information from over 10000 samples. We used publicly available RNA-Seq, mutation and CNV data sets from TCGA. Description and processing details of these data sets are listed at this site: [PanCancer aberrant pathway activity analysis](https://github.com/nvk747/papaa.git).
+
+***Machine learning methodology***
+Logistic regression is a kind of machine learning approach where statistical analysis that is used to predict the outcome of a dependent variable based on prior observations. Changes in gene expression are directly connected to alterations/mutations in genes. We used above approach to predict mutational status given the gene expression. Optimizing to the above prediction of mutational status with gene expression variable, we used elastic net penalty with gradient descent algorithm is used to find the optimal cost function by going over a number of iterations. The objective of the classifier is to determine the probability a given sample (*i*) has a aberrant gene event given the sample’s RNA-Seq measurements (*Xi*). In order to achieve the objective, the classifier learns a vector of coefficients or gene-specific weights (*w*) that optimize the following penalized logistic function.
+
+![Equations for probability measurement ](../../images/aberrant_pi3k_pathway_analysis/equation.png "Equation for prediction of mutational status(Yi) from expression data X(i) for each sample. Mutational status can be estimated by Multiplying Xi with gene specific weights (W). The negative log likelihood (L) is used for calculating minimum weights for each sample")
+
+Where *alpha* and *l* are regularization and elastic net mixing hyperparameters that are only active during training. Each model was tested at multiple *alpha* and *l* values, and 5 fold cross validation was performed.
```

Comment:
This sentence `Each model was tested at multiple *alpha* and *l* values, and 5 fold cross validation was performed` is a bit unclear. It should be - The optimal values of `alpha` and `l` are found using <name_the_hyperparameter_search technique> during 5 fold cross-validation, isn't it? Please make it a bit more clear. Thanks

Reply:
@anuprulez thank you for the changes and appreciate your effort and time. I shall update them and let u know.